robotbase/src/main/java/com/team2357/frc2022/RobotContainer.java

```diff
@@ -70,6 +83,25 @@ public RobotContainer() {
 
     driveSub.setDefaultCommand(new DriveVelocityCommand(driveSub, m_driverControls));
 
+    configureShuffleboard();
+
+    // Setup compressor
+    m_compressor = new Compressor(Constants.CAN_ID.PNEUMATICS_HUB_ID, PneumaticsModuleType.REVPH);
+    m_compressor.enableAnalog(Constants.COMPRESSOR.MIN_PRESSURE_PSI, Constants.COMPRESSOR.MAX_PRESSURE_PSI);
+
+    configureShuffleboard();
+  }
+
+  /**
+   * This method should set up the shuffleboard
+   */
+  public void configureShuffleboard() {
+    DriveTab driveTab = new DriveTab();
+
+    driveTab.addWidget(new FailsafeButtonWidget(SHUFFLEBOARD_TAB_ROBOT, "Drive Failsafe", m_driveSub));
```

Comment:
You need to remove your references to deleted code. This is why the CI gradle build failed.